packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -333,22 +376,34 @@ abstract class FlameChartState<T extends FlameChart, V> extends State<T>
     });
   }
 
-  Future<void> zoomTo(double zoom, {double forceMouseX}) async {
+  Future<void> zoomTo(
+    double zoom, {
+    double forceMouseX,
+    bool jump = false,
+  }) async {
     if (forceMouseX != null) {
       mouseHoverX = forceMouseX;
     }
     await zoomController.animateTo(
       zoom.clamp(FlameChart.minZoomLevel, FlameChart.maxZoomLevel),
-      duration: shortDuration,
+      duration: jump ? Duration.zero : shortDuration,
```

Comment:
may want a very short duration for the scroll wheel case.

Reply:
for consistency used jumpTo in this case as well when `jump` is true. Looks like you do in already in the horizontal case.